bin/kubectl-crossplane-trace

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+function usage {
+  echo "Usage: kubectl crossplane trace KIND NAME [-n| --namespace NAMESPACE] [-h|--help]" >&2 >&2
+  echo "" >&2
+  echo "Traces crossplane object and related objects and prints detailed information" >&2
+  echo "" >&2
+  echo "-h, --help: Print usage" >&2
+}
+
+function check_help {
+  if [ "${1}" == "-h" -o "${1}" == "--help" ] ; then
+    usage
+    exit 1
+  fi
+}
+
+check_help "${1}"
+
+crossplane-cli trace "${@}"
```

Comment:
Where does `crossplane-cli` come from?

Reply:
This will be the built binary of go code. 
e.g.
```
go build -o /bin/crossplane-cli cmd/main.go
```